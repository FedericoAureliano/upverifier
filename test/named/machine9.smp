event eRequest := {request_id: Int}

machine Client
{
    var x : Bool
    ghost y: Bool

    state only {

        on entry do {
            y := x;
            x := true
        }

    }
}

print(Client(only, true, true, true))