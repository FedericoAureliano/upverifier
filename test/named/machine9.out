Client(only, true, true, true)
